Formula/qt-mariadb.rb

```diff
@@ -41,6 +41,11 @@ def install
       system "cmake", ".", *args
       system "cmake", "--build", "."
       system "cmake", "--install", "."
+
+      if OS.linux?
+        sqldrivers_dir = `qmake -query QT_INSTALL_PLUGINS`.strip + "/sqldrivers/"
```

Comment:
```suggestion
        sqldrivers_dir = shell_output("qmake -query QT_INSTALL_PLUGINS").strip + "/sqldrivers/"
```

Reply:
The shell_output() did not work in install function of my linux env.

> Error: An exception occurred within a child process:
>   NoMethodError: undefined method `shell_output' for #<Formulary::FormulaNamespace811c944f1c8265ec4401cac755f35a59::QtMariadb:0x00000000025cc3b0>

Reply:
In the `install` method you need to use `Utils.safe_popen_read`.

What do you expect
```
qmake -query QT_INSTALL_PLUGINS
```
to return here? It seems to me that this is a path that `brew` should know about, so there might not be a need to shell out to `qmake`.

Reply:
It's the dir for Qt to search sql drivers.
Qt SQL drivers need to be installed there, now not so.

> $ qmake -query QT_INSTALL_PLUGINS
> /home/linuxbrew/.linuxbrew/Cellar/qt/6.2.3_1/share/qt/plugins

Reply:
> It's the dir for Qt to search sql drivers.
> 
> Qt SQL drivers need to be installed there, now not so.
> 
> 
> 
> > $ qmake -query QT_INSTALL_PLUGINS
> 
> > /home/linuxbrew/.linuxbrew/Cellar/qt/6.2.3_1/share/qt/plugins

This is a path that belongs to a different formula, so we shouldn't be installing things there.

Reply:
To solve the problem of SQL drivers for Qt such as MariaDB and PostgreSQL not working on Linux, surely the formula of qt.rb should be fixed. The cause is obvious. After resolved, this pull-request can be closed without merging.

Reply:
> To solve the problem of SQL drivers for Qt such as MariaDB and PostgreSQL not working on Linux, surely the formula of qt.rb should be fixed. The cause is obvious.

Could you make a pull request for that fix then? That seems like a better solution